data/countries/CL.yaml

```diff
@@ -12,7 +12,11 @@ holidays:
       - Pacific/Easter
     days:
       01-01:
-        _name: 01-01
+        name: Año Nuevo
```

Comment:
Can we leave "_name: 01-01" here. This would allow other translations for the same day.

Reply:
I'll switch it back